src/Swipeable.js

```diff
@@ -25,8 +25,15 @@ function getPosition(e) {
     : { x: e.clientX, y: e.clientY };
 }
 
+function rotateByAngle({ x, y }, angle) {
+  const angleInRadians = (Math.PI / 180) * angle;
```

Comment:
thoughts on quick out?
```
function rotateByAngle(positions, angle) {
  if (angle === 0) return positions;
```
and don't destructure, since that will be the majority, currently, of use cases?

Reply:
will add that